src/database/DataTypes/TransactionItem.js

```diff
@@ -87,6 +86,47 @@ export class TransactionItem extends Realm.Object {
     // See if any batches can be pruned, i.e. have 0 quantity for this invoice
     this.pruneEmptyBatches(database);
   }
+  /**
+   * Sets Quantity for external supplier invoice, having one Batch per
+   * TransactionLine
+   * @param {int}quantity
+   */
+  setActualBatchQuantity(database, quantity) {
+    const transBatch = this.batches[0];
+    transBatch.packSize = quantity;
```

Comment:
I won't explain, I'll delete, thanks for finding useless function.. it must have been used somewhere, but retired.